include/find_embedding/find_embedding.hpp

```diff
@@ -85,8 +85,8 @@ class parameter_processor {
 
     vector<int> _filter_fixed_vars() {
         vector<int> unscrew(num_vars);
-        assert(var_fixed_unscrewed.size() == num_vars);
-        assert(num_fixed < num_vars);
+        iszero(var_fixed_unscrewed.size() == num_vars);
```

Comment:
This appears to be a source of problems.  Better to change this to `minorminer_assert`